foe-ios/View Controllers/SightingTableViewController.swift

```diff
@@ -19,30 +20,50 @@ class SightingTableViewController: UITableViewController {
     //MARK: Outlets
     @IBOutlet weak var headerView: UIView!
     var activeCell = false
-    var navigationBarOriginalOffset : CGFloat?
     
     var sightings = [Sighting]()
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationBarOriginalOffset = headerView.frame.origin.y
-        statusBarShouldBeHidden = false
-        UIView.animate(withDuration: 0.25) {
-            self.setNeedsStatusBarAppearanceUpdate()
-        }
     }
     
-    func loadStaticData () {
+    func loadStaticPlace () {
```

Comment:
Are these functions are only for testing purposes? If so, could you make a TODO to remind us to remove them later?